checkClock/app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,228 @@
+@import "./variables.scss";
+@import "./clearStyles.scss";
+
+body {
+  font-family: $poppins;
+  background-color: $bg-color;
+  color: $font-color-primary;
+}
+nav {
+  background-color: $color-primary;
+  border-bottom: 1px solid $color-secondary;
+  display: flex;
+  justify-content: space-between;
+  padding: 8px 60px;
+  width: 100%;
+
+  .banner {
+    display: flex;
+    .page-title {
+      font-size: 18px;
+      font-weight: 700;
+      padding: 10px 20px;
+    }
+    .logo {
+      height: 45px;
+      width: 45px;
+      .clock-container {
+        align-items: center;
+        background-color: $color-secondary-light;
+        border-radius: 50%;
+        display: flex;
+        height: 45px;
+        justify-content: center;
+        margin-bottom: 30px;
+        width: 45px;
+        .clock-inside {
+          background-color: $color-secondary;
+          border-radius: inherit;
+          height: 35px;
+          position: relative;
+          width: 35px;
+          .minutes-logo {
+            background-color: $color-secondary-light;
+            border-radius: 10px;
+            border-radius: none;
+            height: 35%;
+            left: 50%;
+            position: absolute;
+            top: 50%;
+            transform-origin: top;
+            transform: translateX(-50%) rotate(180deg);
+            width: 3px;
+          }
+          .hours-logo {
+            background-color: $color-primary;
+            border-radius: 10px;
+            height: 27%;
+            left: 50%;
+            position: absolute;
+            top: 50%;
+            transform-origin: top;
+            transform: translateX(-50%) rotate(180deg);
+            width: 3px;
+          }
+        }
+      }
+    }
+  }
+
+  .navigation {
+    background-color: $color-secondary;
+    border-radius: 15px;
+    font-family: "Poppins", sans-serif;
```

Comment:
You can also create specific variables for your fonts, same like yours colors.

Reply:
Fixed